src/main/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/CompanyNumberInterceptor.java

```diff
@@ -32,7 +32,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         final boolean companyNumberMatches = doCompanyNumbersMatch(companyNumber, objection);
 
         if (!companyNumberMatches) {
-            apiLogger.debugContext(requestId, "Provided company number does not match objection company number");
+            apiLogger.errorContext(requestId, "Provided company number does not match objection company number", null);
```

Comment:
There was nothing logged when eg. company number was incorrect for url so I've updated the error condition logging to errorContext. This was already the case for the ObjectionStatus.

Reply:
Imo this isn't an error scenario? Maybe worth bumping it from debug to info, but I don't think this deserves an error log unless an exception being thrown